spec/models/last_updated_at_spec.rb

```diff
@@ -46,54 +46,6 @@
     end
   end
 
-  it 'updates the application_choices when a application_work_history_break is added' do
```

Comment:
These specs are no longer needed if we don't want to update the application choice when updating the application form work experiences. 

This will do the touch correctly on application_form or choice
https://github.com/DFE-Digital/apply-for-teacher-training/blob/05771b2f601f559070580d23f12a8e560d406558/app/models/application_experience.rb#L3